cg/store/models.py

```diff
@@ -1152,7 +1152,7 @@ class IlluminaSampleSequencingMetrics(SampleRunMetrics):
 class PacBioSampleSequencingMetrics(SampleRunMetrics):
     """Sequencing metrics for a sample sequenced on a PacBio instrument. The metrics are per sample, per cell."""
 
-    __tablename__ = "pacbio_sample_sequencing_metrics"
+    __tablename__ = "pacbio_sample_run_metrics"
```

Comment:
Is this reflected in the database?

Reply:
yes, it was actually not working because the names didn't match